.github/workflows/tests.yml

```
@@ -0,0 +1,78 @@
+      website: ${{ steps.filter.outputs.website }}
+      typegate: ${{ steps.filter.outputs.typegate }}
+      full: ${{ steps.filter.outputs.full }}
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dsherret/rust-toolchain-file@v1
+      - if: ${{ env.AWS_ACCESS_KEY_ID != '' }}
+        name: Setup sccache secrets
+        run: echo "RUSTC_WRAPPER=sccache" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:33: Double quote to prevent globbing and word splitting

(shellcheck)